doc-content/enterprise-only/processes/interacting-with-processes-process-instances-default-filters-proc.adoc

```diff
@@ -0,0 +1,15 @@
+[id='interacting-with-processes-process-instances-default-filters-proc']
+= Filtering process instances using default filters
+
+You can set a process instance filter as a default filter using the *Saved Filter* option in {CENTRAL}.
```

Comment:
Perhaps add the default filter function after this phrase. Something like: A default filter is the one that will be executed every time the page is open by the user.

Reply:
Ack